source/our-stacks/back-end/index.md

```diff
@@ -1,6 +1,80 @@
----
-title: Back-end
-date: 2016-09-29 15:27:11
-layout: stack
----
-Total Back-end
\ No newline at end of file
+#Backend
+The separation of software systems into front and back ends simplifies development and separates maintenance. A rule of thumb is that the front (or "client") side is any component manipulated by the user. The server-side (or "back end") code resides on the server.
+
+###Product
+Backend developers in Nodes build APIs with documentation & Backends 
```

Comment:
"Backend developers ... build ... backends" sounds a bit non-descriptive :D Would substituting the last backends for something like content and user management systems work?